client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -354,6 +356,80 @@ class EmailProvidersComparison extends Component {
 		);
 	}
 
+	getAvailableDiscountForGoogle(
+		currencyCode,
+		isEligibleForFreeTrial,
+		gSuiteProduct,
+		productIsDiscounted,
+		standardPrice
+	) {
+		const { translate } = this.props;
+		if ( productIsDiscounted ) {
```

Comment:
Blank line please 😉. As [our guidelines](https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/javascript.md) says:

> Use spacing liberally throughout your code. “When in doubt, space it out.”

Reply:
“When in doubt, space it out.” 😆